src/screens/DashboardScreen.tsx

```diff
@@ -68,6 +70,24 @@ const PrecinctsScreen = ({ isScanning, status }: Props) => {
                         <small>{shortDateTime(batch.endedAt)}</small>
                       )}
                     </TD>
+                    <td>
+                      <Button
+                        onClick={() => {
+                          if (
+                            // eslint-disable-next-line no-restricted-globals
+                            confirm(
```

Comment:
This was the dumbest thing I could think of that would make it work. @beausmith you cool with using `confirm` or do we have a better modal thing we can use?

Reply:
can I be so bold as to suggest we merge this and then file a follow-up issue to improve? MVP :)